metrics/internal/version/version.go

```diff
@@ -0,0 +1,8 @@
+package version
+
+const version = "0.0.1"
```

Comment:
Can do. But, weren't we also talking about getting rid of the version in operator?

And, since exporter is a separate component (like rook noobaa csi etc) used by OCS, can't it have a different version?

I'm fine with using any version that aligns to our practices.

Reply:
I wanted to get rid of the `Spec.Status` field, yes, but the operator binary itself should have some notion of identity as to which version it is. It may still make sense to have a field like `Status.LastReconcilerVersion`. I think since this binary is just another part of the same project, it makes sense to have it match the version of the project/operator.